### PR TITLE
added 'create account successful' test case.

### PR DESCRIPTION
**Pull request by korkoza**
1. Simplified test case #1. As it's not possible to get text from input elements
2. Added browser.restart before waitForAngular in onPrepare. And it works
3. Changed names of methods
4. Finished automation of 'Create account successfully' test case
5. Renamed specs according to test cases from document

### CODE REVIEW

File: specs/02_create_acc_neg.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
+let RegisterPage = require('../pages/registerPage');
+let MainPage = require('../pages/mainPage');
+let LoginPage = require('../pages/loginPage');
 
 describe('Login page', () => {
     beforeAll(async () => {
-        await browser.restart();
         await browser.manage().setTimeouts({ implicit: browser.params.implicitWait });
-        browser.waitForAngularEnabled(false);
         await browser.get('https://hotline.ua/');
```
Comment (nbabii) [inline specs/02_create_acc_neg.js]: Please move `await browser.get('https://hotline.ua/');` to the main page and create method `open()`

File: pages/RegisterPage.js [hunk 2]
```diff
@@ -34,30 +38,35 @@ class RegisterPage extends BasePage {
         return new TextViewElement(element(by.css(".field.error[type='password'] + div div")), "Password Error");
     }
 
-    getSubmitReg() {
-        return new ButtonElement(element(by.id("submit-button")), "Create Submit registration button");
+    getSubmit() {
```
Comment (nbabii) [inline pages/RegisterPage.js]: rename to `getSubmitBtnElement`

File: specs/01_create_acc_pos.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable max-len */
+let RegisterPage = require('../pages/registerPage');
+let MainPage = require('../pages/mainPage');
+let LoginPage = require('../pages/loginPage');
+
+describe('Login page', () => {
+    beforeAll(async () => {
+        await browser.manage().setTimeouts({ implicit: browser.params.implicitWait });
+        await browser.get('https://hotline.ua/');
```
Comment (nbabii) [inline specs/01_create_acc_pos.js]: Please move `await browser.get('https://hotline.ua/');` to the main page and create method `open()`

File: specs/01_create_acc_pos.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        await MainPage.navigateToSignIn();
+        expect(await LoginPage.getSignUpElement()).toBeTruthy();
+
+        // step 2
+        await LoginPage.navigateToSignUp();
+        expect(await browser.getTitle()).toEqual('Регистрация нового пользователя');
+
+        // step 3
+        let rndEmail = `protractor.automation01+${Math.floor(Math.random() * 10000)}@gmail.com`;
+        console.log(rndEmail);
```
Comment (nbabii) [inline specs/01_create_acc_pos.js]: please remove extra `console.log`

File: specs/01_create_acc_pos.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+        let rndNick = `protractor.automation01${Math.floor(Math.random() * 10000)}`;
+        console.log(rndNick);
```
Comment (nbabii) [inline specs/01_create_acc_pos.js]: please remove extra `console.log`

File: specs/02_create_acc_neg.js [hunk 6]
```diff
@@ -1,13 +1,11 @@
...
-let MainPage = require('../pages/MainPage');
-let LoginPage = require('../pages/LoginPage');
+let RegisterPage = require('../pages/registerPage');
+let MainPage = require('../pages/mainPage');
+let LoginPage = require('../pages/loginPage');
 
 describe('Login page', () => {
     beforeAll(async () => {
-        await browser.restart();
         await browser.manage().setTimeouts({ implicit: browser.params.implicitWait });
```
Comment (nbabii) [inline specs/02_create_acc_neg.js]: what if you move `await browser.manage().setTimeouts({ implicit: browser.params.implicitWait });` also to onPrepare after browser restart

File: specs/03_succes_login.js [hunk 7]
```diff
@@ -1,11 +1,9 @@
-let LoginPage = require('../pages/LoginPage');
-let MainPage = require('../pages/MainPage');
+let LoginPage = require('../pages/loginPage');
+let MainPage = require('../pages/mainPage');
 
 describe('Login page', () => {
     beforeAll(async () => {
-        await browser.restart();
         await browser.manage().setTimeouts({ implicit: browser.params.implicitWait });
```
Comment (nbabii) [inline specs/03_succes_login.js]: what if you move `await browser.manage().setTimeouts({ implicit: browser.params.implicitWait });` also to onPrepare after browser restart

File: specs/01_create_acc_pos.js [hunk 8]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable max-len */
+let RegisterPage = require('../pages/registerPage');
+let MainPage = require('../pages/mainPage');
+let LoginPage = require('../pages/loginPage');
+
+describe('Login page', () => {
+    beforeAll(async () => {
+        await browser.manage().setTimeouts({ implicit: browser.params.implicitWait });
```
Comment (nbabii) [inline specs/01_create_acc_pos.js]: what if you move `await browser.manage().setTimeouts({ implicit: browser.params.implicitWait });` also to onPrepare after browser restart

File: specs/03_succes_login.js [hunk 9]
```diff
@@ -1,11 +1,9 @@
...
-        browser.waitForAngularEnabled(false);
         await browser.get('https://hotline.ua/');
```
Comment (nbabii) [inline specs/03_succes_login.js]: same here, move to method `open()`

File: specs/02_create_acc_neg.js (hunk 1 above)
Comment (korkoza) [reply specs/02_create_acc_neg.js]: fixed

File: pages/RegisterPage.js (hunk 2 above)
Comment (korkoza) [reply pages/RegisterPage.js]: fixed

File: specs/01_create_acc_pos.js (hunk 3 above)
Comment (korkoza) [reply specs/01_create_acc_pos.js]: fixed

File: specs/01_create_acc_pos.js (hunk 4 above)
Comment (korkoza) [reply specs/01_create_acc_pos.js]: fixed

File: specs/01_create_acc_pos.js (hunk 5 above)
Comment (korkoza) [reply specs/01_create_acc_pos.js]: fixed

File: specs/02_create_acc_neg.js (hunk 6 above)
Comment (korkoza) [reply specs/02_create_acc_neg.js]: it works correctly. I tested by breaking css locator of getPageBaseElement element and as a result time for waiting was a bit more then 10 seconds

File: specs/03_succes_login.js (hunk 7 above)
Comment (korkoza) [reply specs/03_succes_login.js]: it works correctly. I tested by breaking css locator of getPageBaseElement element and as a result time for waiting was a bit more then 10 seconds

File: specs/01_create_acc_pos.js (hunk 8 above)
Comment (korkoza) [reply specs/01_create_acc_pos.js]: it works correctly. I tested by breaking css locator of getPageBaseElement element and as a result time for waiting was a bit more then 10 seconds

File: specs/03_succes_login.js (hunk 9 above)
Comment (korkoza) [reply specs/03_succes_login.js]: fixed

**Pull request merged**